src/cryptoadvance/specter/server_endpoints/wallets_api.py

```diff
@@ -0,0 +1,933 @@
+    txlist = json.loads(
+        process_txlist(
+            txlist, idx=0, limit=0, search=search, sortby=sortby, sortdir=sortdir
+        )["txlist"]
+    )
+    includePricesHistory = request.args.get("exportPrices", "false") == "true"
+
+    # stream the response as the data is generated
+    response = Response(
+        txlist_to_csv(wallet, txlist, app.specter, current_user, includePricesHistory),
+        mimetype="text/csv",
+    )
+    # add a filename
+    response.headers.set(
+        "Content-Disposition", "attachment", filename="transactions.csv"
+    )
+    return response
+
+
+# Export wallet UTXO list
+@wallets_endpoint_api.route("/wallet/<wallet_alias>/utxo.csv")
+@login_required
+def utxo_csv(wallet_alias):
+    try:
+        wallet = app.specter.wallet_manager.get_by_alias(wallet_alias)
+        includePricesHistory = request.args.get("exportPrices", "false") == "true"
+        search = request.args.get("search", None)
+        sortby = request.args.get("sortby", "time")
+        sortdir = request.args.get("sortdir", "desc")
+        txlist = json.loads(
+            process_txlist(
+                wallet.full_utxo,
+                idx=0,
+                limit=0,
+                search=search,
+                sortby=sortby,
+                sortdir=sortdir,
+            )["txlist"]
+        )
+        # stream the response as the data is generated
+        response = Response(
+            txlist_to_csv(
+                wallet,
+                txlist,
+                app.specter,
+                current_user,
+                includePricesHistory,
+            ),
+            mimetype="text/csv",
+        )
+        # add a filename
+        response.headers.set("Content-Disposition", "attachment", filename="utxo.csv")
+        return response
+    except Exception as e:
+        handle_exception(e)
+        return _("Failed to export wallet utxo. Error: {}").format(e), 500
+
+
+@wallets_endpoint_api.route("/wallet/<wallet_alias>/asset_balances")
+@login_required
+def asset_balances(wallet_alias):
+    try:
+        wallet = app.specter.wallet_manager.get_by_alias(wallet_alias)
+        if app.specter.is_testnet:
+            label = "tBTC"
+        elif app.specter.is_liquid:
+            label = "LBTC"
+        else:
+            label = "BTC"
+
+        amounts = []
+        textUnit = app.specter.unit
+        asset_balances = {
+            "btc": {
+                "balance": wallet.full_available_balance,
+                "label": label,
+            },
+            "sat": {
+                "balance": int(wallet.full_available_balance * 1e8),
+                "label": "sat",
+            },
+        }
+        for asset in wallet.balance.get("assets", {}).keys():
+            asset_balances["asset"] = {
+                "balance": wallet.balance.get("assets", {})
+                .get(asset, {})
+                .get("trusted", 0),
+                "label": "{{{% endfor %} asset | assetlabel }}",
```

Comment:
This jinja content can't make sense here, can it?

Reply:
Definitely. Fixed.